doc-rosindex.yaml

```diff
@@ -0,0 +1,16 @@
+%YAML 1.1
+# ROS buildfarm doc-build file
+---
+documentation_type: docker_build
+doc_repositories:
+- https://github.com/ros2/rosindex.git
+jenkins_job_priority: 90
+jenkins_job_timeout: 120
+targets:
+  ubuntu:
+    trusty:
```

Comment:
Is there a reason to run this job on Trusty? If not I would prefer it to run on a more up-to-date distro to avoid it using a platform which will be EOL in less then half a year.

Reply:
If this part isn't being used than the build file shouldn't need to provide it.

Reply:
Ok, optional it is. I'll update ros-infrastructure/ros_buildfarm#576.